src/test/streaming_test.cpp

```diff
@@ -449,6 +451,228 @@ node {
     ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::OK);
 }
 
+// Generative AI case + automatic timestamping server-side - Python
+
+#include <pybind11/embed.h>  // everything needed for embedding
+namespace py = pybind11;
+#include "../python/python_backend.hpp"
+
+TEST_F(StreamingTest, SingleStreamSend1Receive3Python) {
+    PythonInterpreterModule pythonModule;
+    pythonModule.start(ovms::Config::instance());
```

Comment:
Do we need to use singleton here? Maybe we could initialize Config in the test scope?

Reply:
There's no public constructor for Config. Also, in that case it's only needed to start python module.